lib/timeline/component/item/RangeItem.js

```diff
@@ -14,7 +14,7 @@ class RangeItem extends Item {
  *                                  // TODO: describe options
  */
   constructor(data, conversion, options) {
-    super()
+    super();
```

Comment:
It is not a goo style to change code that has nothing to do with this RP. Could you please remove this change from this RP. Maybe we will get a code-style fixer in the future that helps with this!